serow_ros/src/serow_driver.cpp

```diff
@@ -81,8 +89,9 @@ class SerowDriver {
     }
 
     void force_torque_state_topic_callback(const geometry_msgs::WrenchStamped::ConstPtr& msg) {
-        std::string_view frame_id = msg->header.frame_id;
-        ft_data_[frame_id].push(*msg);
```

Comment:
this should have initialized `ft_data_` for every `frame_id` so you don't need the foot frames parameters.